.github/workflows/audit.yml

```diff
@@ -13,18 +13,12 @@ permissions:
   contents: read
 
 jobs:
-  security-audit:
+  cargo-deny:
     permissions:
-      checks: write  # for rustsec/audit-check to create check
-      contents: read  # for actions/checkout to fetch code
-      issues: write  # for rustsec/audit-check to create issues
+      checks: write
+      contents: read
+      issues: write
```

Comment:
EmbarkStudios/cargo-deny-action actions do not seem create issues (https://github.com/EmbarkStudios/cargo-deny-action/issues/64), so these permissions are probably not needed.

Reply:
I mean, not like the rustsec one has ever created an issue ...

Reply:
(The last time rustsec action created an issue in this repository was two years ago, which IIRC was before fine-grained permission settings in GitHub actions were supported.)
https://github.com/tokio-rs/tokio/issues?q=is%3Aissue+author%3Aapp%2Fgithub-actions